http2/src/main/java/com/blazemeter/jmeter/http2/sampler/HTTP2SampleResult.java

```diff
@@ -44,61 +17,33 @@
 import org.apache.jmeter.threads.ListenerNotifier;
 import org.apache.jmeter.threads.SamplePackage;
 import org.eclipse.jetty.http.HttpFields;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-/**
- * This is a specialisation of the SampleResult class for the HTTP protocol.
- */
 public class HTTP2SampleResult extends HTTPSampleResult {
 
-  private static final long serialVersionUID = 241L;
-
-  /**
-   * Set of all HTTP methods, that have no body
-   */
-  private static final Set<String> METHODS_WITHOUT_BODY = new HashSet<>(
-      Arrays.asList(HTTPConstants.HEAD, HTTPConstants.OPTIONS, HTTPConstants.TRACE));
-
-  private static int idCount = 0;
-
-  private int id;
-
+  static final String HTTP2_RESPONSE_CODE_4 = "Not Found";

```

Comment:
Is this visibility ok? is the constant used in this class? if not move to proper class where it is being used, if yes then make them private or public.